README.md

```diff
@@ -11,7 +11,7 @@ cd ~/Projects/LLVM
 
 git clone http://llvm.org/git/llvm.git
 cd llvm
-git checkout 106bb1fe2db88fe77458ec594c461c8428588dab
+git checkout dc30ded6fb9526aba31a86b97e8f69fc9baece00
```

Comment:
These changes should be here since you rebased from master which has these strings updated, no?